math/vecops/inc/ROOT/RVec.hxx

```diff
@@ -3135,6 +3135,38 @@ inline RVec<std::size_t> Range(std::size_t begin, std::size_t end)
    return ret;
 }
 
+/// Produce RVec<int> with entries equal to begin, begin+stride, ..., end +/- stride.
+ /// An empty RVec is returned if begin >= end and stride > 0 or if
+ /// begin < end and stride < 0. Allows for negative begin, end, and/or
+ /// stride
+ inline RVec<int> Range(int begin, int end, int stride)
```

Comment:
I understand the need for signed integers since the stride can be negative, but `int` is definitely too restrictive:
```suggestion
 inline RVec<long long int> Range(long long int begin, long long int end, long long int stride)
```